interfaces/builtin/opengl.go

```diff
@@ -121,6 +125,7 @@ var openglConnectedPlugUDev = []string{
 	`KERNEL=="renderD[0-9]*"`,
 	`KERNEL=="nvhost-*"`,
 	`KERNEL=="nvmap"`,
+	`KERNEL=="tegra_dc_*"`,
```

Comment:
Did you see any additional devices in /dev that snap-confine's Nvidia code needs to support?

Reply:
No, fortunately these devices do show in /sys and have proper udev properties.